File: Hackathon.UI/src/app/models/notifications/NotificationType.ts
export enum NotificationType {
  System = 0,
  TeamInvite = 1,
  EventInvite = 2,

  /* Принятие в команду */
  TeamAcceptance = 3,

  /* Отклонение запроса на присоединения к команде */
  JoinTeamRejection = 4,

  /* Запрос дружбы */
  FriendshipRequest = 5,

  /* Запрос дружбы принят */
  FriendshipAccepted = 6,

  /* Запрос дружбы отклонен */
  FriendshipRejected = 7,

  /* Мероприятие скоро начнется */
  EventWillStartSoon = 8,

  /* Новое сообщение в чате команды */
  NewTeamChatMessage = 9,

  /* Новое сообщение в чате мероприятия*/
  NewEventChatMessage = 10,
}

File: Hackathon.Informing.Abstractions/Models/NotificationType.cs
namespace Hackathon.Informing.Abstractions.Models;

/// <summary>
/// Тип уведомления
/// </summary>
public enum NotificationType
{
    /// <summary>
    /// Системное уведомление
    /// </summary>
    System = 0,

    /// <summary>
    /// Приглашение в команду
    /// </summary>
    TeamInvite = 1,

    /// <summary>
    /// Приглашение на мероприятие
    /// </summary>
    EventInvite = 2
}
